node-notes-cli/app.js

```diff
@@ -0,0 +1,32 @@
+const dataJson = require('./data.json');
+
+const fs = require('fs');
+
+if (process.argv[2] === 'read') {
+  for (const key in dataJson.notes) {
+    console.log(`${key}: ${dataJson.notes[key]}`);
+  }
+}
+
+if (process.argv[2] === 'create') {
+  dataJson.notes[dataJson.nextId] = process.argv[3];
+  dataJson.nextId = parseInt(Object.keys(dataJson.notes)[Object.keys(dataJson.notes).length - 1]) + 1;
```

Comment:
The id and the number of notes available are not actually related. You can simply increment `dataJson.nextId` with `++`.